sphinx-docs/usermanual/userman_main.rst

```diff
@@ -0,0 +1,47 @@
+User Manual
+============
+
+If you know which user manual you'd like to see, click to jump to a specific  manual:
+
+* :doc:`userman_admin`
```

Comment:
By the way, might be better to use a toctree directive here instead of a bunch of `:doc:` links.